web-client/src/presenter/actions/broadcastLogoutAction.test.ts

```diff
@@ -6,25 +6,7 @@ import { runAction } from '@web-client/presenter/test.cerebral';
 presenter.providers.applicationContext = applicationContext;
 
 describe('broadcastLogoutAction', () => {
-  it('does not broadcast an event if skipBroadcast is true and CI is undefined', async () => {
-    delete process.env.CI;
-
-    await runAction(broadcastLogoutAction, {
-      modules: {
-        presenter,
-      },
-      props: {
-        skipBroadcast: true,
-      },
-    });
-
-    expect(
-      applicationContext.getBroadcastGateway().postMessage,
-    ).not.toHaveBeenCalled();
-  });
-
-  it('does not broadcast an event if skipBroadcast is true and CI is defined', async () => {
-    process.env.CI = 'true';
```

Comment:
See [my comment on `broadCastLogoutAction` below](https://github.com/ustaxcourt/ef-cms/pull/5176/files#r1695796882). We should not longer need to test anything to do with `process.env.CI` being true/false.